### PR TITLE
Update scroll fix to include iOS8 and greater

### PR DESCRIPTION
**Pull request by fractaltheory**
Status: **Ready for Review**
Reviewers: @donnielrt @haroldtreen 
CC: @jansepar @johnboxall @mobify-derrick 

## Changes
- Updates user agent regular expression to match version of iOS 8 **and up**
- Changes function names to better reflect scope of fix
- Fix test to refer to new function name
- Add a test for the function that lets us know if user agent is iOS8 or greater

## Jira Tickets:
- [RTM-340](https://mobify.atlassian.net/browse/RTM-340) 

## Todos:
- [x] Engineer +2
- [ ] npm release

### Feedback:
- [x] Change package name to mobify-capturejs
- [x] Add test for `isIOS8OrGreater` method

## How to Test
- Update version of capturejs in your Adaptive.js 1.x `package.json` file to point to this branch, i.e. `mobify/capturejs#ios9-scroll-fix`
- Run `grunt preview` in your Adaptive.js 1.x project
- Test on an affected device (i.e. iOS 8+) and ensure the scroll bug does not occur while navigating the site

### CODE REVIEW

**Comment by jansepar** [comment]
So, according to Donnie, the issue with the app banner is no longer an issue, but we were using that "isiOS8" method to fix it for iOS8. We probably only want to apply the app banner fix to iOS8.

**Comment by fractaltheory** [comment]
@jansepar fair enough - I put back the original isIOS8_0 function and it's used for the smart banner fix now: https://github.com/mobify/capturejs/commit/ba586a6371e7b47c8f2f61e22f9dae809e6a2c2c

**Comment by donnielrt** [comment]
@fractaltheory fixed the smart banner test

File: src/capture.js [hunk 1]
```diff
@@ -389,6 +389,13 @@ Capture.isIOS8_0 = function() {
     return IOS8_REGEX.test(window.navigator.userAgent);
 };
 
+var IOS_REGEX = /ip(?:hone|od|ad).*Version\/(\d){1,2}\.\d/i;
+Capture.isIOS8OrGreater = function() {
+    var match = window.navigator.userAgent.match(IOS_REGEX);
+
+    return match ? match[1] >= 8 : false;
```
Comment (jansepar) [inline src/capture.js]: Could we add a test for this? Would be good to assert that this returns true for an iOS8 and 9 UA, and false for an iOS7.

**Comment by jansepar** [comment]
Looks pretty legit! :+1: once that test is written :) I ran the tests on Saucelabs, no regressions (although looks like Saucelabs doesn't support testing on iOS 4.3 anymore :P)

**Comment by fractaltheory** [comment]
@jansepar added a test: https://github.com/mobify/capturejs/commit/3dc84bab7581913f6a33dafde81351076752d252 what do you think?

File: CHANGELOG [hunk 2]
```diff
@@ -1,3 +1,8 @@
+1.0.4
```
Comment (donnielrt) [inline CHANGELOG]: Kinda weird that this is the frist version we're publishing, but (shrug)

**Comment by donnielrt** [comment]
Verbal +1 from Shawn.

**Pull request merged**